packages/yoga/src/Avatar/native/Placeholder.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+import Icon from '../../Icon';
+
+const Placeholder = ({ fill, icon }) => {
+  return <Icon as={icon} width="50%" height="50%" fill={fill || 'white'} />;
```

Comment:
Could `white` be used as the default value of `fill`?

```js
const Placeholder = ({ fill = 'white', icon }) => {
```

OR

```js
Placeholder.defaultProps = {
  fill: 'white',
};
```

Reply:
@leonardokl Done!